pai-management/k8sPaiLibrary/maintainlib/common.py

```diff
@@ -56,6 +56,23 @@ def execute_shell(shell_cmd, error_msg):
 
 
 
+def execute_shell_retry(shell_cmd, clean_cmd, error_msg):
+    count = 0
+    success_flag = False
+    while count < 5 and not success_flag:
+        try:
+            subprocess.check_call( shell_cmd, shell=True )
+            success_flag = True
+        except subprocess.CalledProcessError:
+            count += 1
+            logger.error(error_msg)
+            logger.info("%s error, retrying %d", shell_cmd, count)
+            logger.info("Cleaning failed Pods...")
```

Comment:
This function could be reused in other module too. Please remove the specific infomation.
Or, you shouldn't put it in the comm.py module.